frontend/components/CustomBurgerIcon/CustomBurgerIcon.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import getConfig from '../../helpers/getConfig';
+
+const { BurgerSvg } = getConfig();
+
+/**
+ * The home icon component.
+ * @param {Object} props The icon component properties.
+ * @returns {JSX}
+ */
+const CustomBurgerIcon = props => (
+  <div {...props} dangerouslySetInnerHTML={{ __html: BurgerSvg }} />
```

Comment:
We should avoid using dangerouslySetInnerHTML. Try using the Icon component which can be imported ``` import { Icon } from '@shopgate/engage/components'```

Reply:
Talked to @Carsten89  about that. We don't use the Icon component, because we want to be able to put the whole SVG in the config. 

If we use the Icon component, we have to separate the SVG into content and viewBox.